data-otservbr-global/lib/core/storages.lua

```diff
@@ -2392,51 +2235,261 @@ Storage = {
 			HuntingForTokens = {},
 			OpticordingSphere = {},
 			SmallSapphire = {},
-			TheGravediggerOfDrefia = {},
+			TheGravediggerOfDrefia = {
+				QuestStart = 44200,
+				Mission01 = 44201,
+				Mission02 = 44203,
+				Mission03 = 44204,
+				Mission04 = 44205,
+				Mission05 = 44206,
+				Mission06 = 44207,
+				Mission07 = 44208,
+				Mission08 = 44209,
+				Mission09 = 44210,
+				Mission10 = 44211,
+				Mission11 = 44212,
+				Mission12 = 44213,
+				Mission13 = 44214,
+				Mission14 = 44215,
+				Mission15 = 44216,
+				Mission16 = 44217,
+				Mission17 = 44218,
+				Mission18 = 44219,
+				Mission19 = 44220,
+				Mission20 = 44221,
+				Mission21 = 44222,
+				Mission22 = 44223,
+				Mission23 = 44224,
+				Mission24 = 44225,
+				Mission25 = 44226,
+				Mission26 = 44227,
+				Mission27 = 44228,
+				Mission28 = 44229,
+				Mission29 = 44230,
+				Mission30 = 44231,
+				Mission31 = 44232,
+				Mission32 = 44233,
+				Mission32a = 44234,
+				Mission32b = 44235,
+				Mission33 = 44236,
+				Mission34 = 44237,
+				Mission35 = 44238,
+				Mission36 = 44239,
+				Mission36a = 44240,
+				Mission37 = 44241,
+				Mission38 = 44242,
+				Mission38a = 44243,
+				Mission38b = 44244,
+				Mission38c = 44245,
+				Mission39 = 44246,
+				Mission40 = 44247,
+				Mission41 = 44248,
+				Mission42 = 44249,
+				Mission43 = 44250,
+				Mission44 = 44251,
+				Mission45 = 44252,
+				Mission46 = 44253,
+				Mission47 = 44254,
+				Mission48 = 44255,
+				Mission49 = 44256,
+				Mission50 = 44257,
+				Mission51 = 44258,
+				Mission52 = 44259,
+				Mission53 = 44260,
+				Mission54 = 44261,
+				Mission55 = 44262,
+				Mission56 = 44263,
+				Mission57 = 44264,
+				Mission58 = 44265,
+				Mission59 = 44266,
+				Mission60 = 44267,
+				Mission61 = 44268,
+				Mission62 = 44269,
+				Mission63 = 44270,
+				Mission64 = 44271,
+				Mission65 = 44272,
+				Mission66 = 44273,
+				Mission67 = 44274,
+				Mission68 = 44275,
+				Mission69 = 44276,
+				Mission70 = 44277,
+				Mission71 = 44278,
+				Mission72 = 44279,
+				Mission73 = 44280,
+				Mission74 = 44281,
+				Mission75 = 44282,
+				Mission76 = 44283,
+				Bookcase = 44285,
+				SmallDragonTears1 = 44286,
+				SmallDragonTears2 = 44287,
+				SmallDragonTears3 = 44288,
+			},
 			TheRepenters = {},
 		},
 		U10_20 = { -- update 10.20 - Reserved Storages 44301 - 44400
-			CaveExplorerOutfits = {},
-			SpikeTasks = {},
+			SpikeTaskQuest = {
+				QuestLine = 44301,
+				Gnomilly = 44302,
+				Gnombold = {
+					Points = 44303,
+					Nests = 44304,
+					Fertilise = 44305,
+					Kill = 44306,
+					Charges = 44307,
+				},
+				Gnomargery = {
+					Points = 44308,
+					Deliver = 44309,
+					Undercover = 44310,
+					Temperature = 44311,
+					Kill = 44312,
+				},
+				Constants = {
```

Comment:
I think the naming standard should be maintained and I also believe that they are not constant.